walkthroughs/1C-managing-and-protecting-using-api-mgmt/walkthrough.adoc

```diff
@@ -0,0 +1,381 @@
+
+=== Add the Order API to Red Hat 3scale
+
+. From the *Dashboard*, select the *New API* item.
+. Select the *Define Manually* option.
+. Enter `orders-api` as the *Name* and *System name*:
+. Leave the *Description* field empty.
+. Click *Add API* at the bottom of the screen. The Order API service will be created and you will be redirected to the *Overview* page.
+
+=== Configure integration with Orders API
+
+. Configure OpenID Connect as authentication method for the API:
+.. Click *Integration > Configuration* from the side navigation.
+.. Select *edit integration settings* in the top right corner.
+.. In the *Authentication* section at the bottom of the screen select *OpenID Connect*.
+.. Click *Update Service*.
+.. When prompted for confirmation, click *OK*.
+
+. Configure the integration with the Orders API:
+.. If you are not already on the *Configuration* page, click *Integration > Configuration* from the side navigation.
+.. Click *add the base URL of your API and save the configuration.*
+.. In the *Private Base URL* field, enter the URL of the *Order Entry System* from the previous Solution Pattern that should look like this: `https://order-entry-ui-<NAMESPACE>.{openshift-app-host}`.
+.. Keep the fields *Staging Public Base URL* and *Production Public Base URL* unchanged, their values should look as follows:
++
+[subs="attributes+"]
+----
+https://orders-api-{user-sanitized-username}-apicast-staging.{openshift-app-host}
+
+https://orders-api-{user-sanitized-username}-apicast-production.{openshift-app-host}
+----
++
+.. Expand the *Authentication Settings* and make sure `Red Hat Single Sign-On` is selected in the *OpenID Connect Issuer Type*.
+.. In *OpenID Connect Issuer*, enter:
++
+[subs="attributes+"]
+----
+https://{zync-client-id}:ZYNC_SECRET@sso-user-sso.{openshift-app-host}/auth/realms/{user-sanitized-username}
+----
+Replace `ZYNC_SECRET` with the value of the Client Secret of the `{zync-client-id}` client created previously in {sso-ProductName}.
++
+.. In *OIDC Authorization Flow*, keep the `Authorization Code Flow` checkbox enabled.
+.. In the *Secret Token* field *OIDC Authorization Flow*, enter:
++
+[subs="attributes+"]
+----
+{gateway-secret-token}
+----
+.. In the *Credentials location*, select "As HTTP Headers" radio button.
+.. Add CORS (Cross-Origin-Resource-Sharing) policy, which is needed to call the API from the Developer Portal.
+... Expand the *Policies* section.
+... Select *Add Policy*.
+... Select *CORS* from the list.
+... Click on the arrow on the right of the policy name and drag the *CORS* policy to place it on the top of the Policy Chain.
+.. Click *Update the Staging Environment*.
+
+[type=verification]
+Was the configuration saved successfully without any errors?
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above. If you are still having issues, contact your administrator.
+
+=== Configure an Application Plan and an Application
+
+. Create a new *Application Plan*:
```

Comment:
Should there be a brief paragraph above this explaining an "application plan is"?